new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItSimpleNginxValidation.java

```diff
@@ -0,0 +1,620 @@
+    logger.info("Check for admin server pod {0} existence in namespace {1}",
+        adminServerPodName, domainNamespace);
+    checkPodCreated(adminServerPodName);
+
+    // check managed server pods exists
+    for (int i = 1; i <= replicaCount; i++) {
+      logger.info("Check for managed server pod {0} existence in namespace {1}",
+          managedServerPrefix + i, domainNamespace);
+      checkPodCreated(managedServerPrefix + i);
+    }
+
+    // check admin server pod is ready
+    logger.info("Wait for admin server pod {0} to be ready in namespace {1}",
+        adminServerPodName, domainNamespace);
+    checkPodReady(adminServerPodName);
+
+    // check managed server pods are ready
+    for (int i = 1; i <= replicaCount; i++) {
+      logger.info("Wait for managed server pod {0} to be ready in namespace {1}",
+          managedServerPrefix + i, domainNamespace);
+      checkPodReady(managedServerPrefix + i);
+    }
+
+    logger.info("Check admin service {0} is created in namespace {1}",
+        adminServerPodName, domainNamespace);
+    checkServiceCreated(adminServerPodName);
+
+    // check managed server services created
+    for (int i = 1; i <= replicaCount; i++) {
+      logger.info("Check managed server service {0} is created in namespace {1}",
+          managedServerPrefix + i, domainNamespace);
+      checkServiceCreated(managedServerPrefix + i);
+    }
+  }
+
+  @Test
+  @Order(2)
+  @DisplayName("Create an ingress for a WebLogic domain in the specified domain namespace")
+  public void testCreateIngress() {
+
+    // create an ingress in domain namespace
+    assertThat(assertDoesNotThrow(() -> createIngress(domainNamespace, domainUid, clusterName, managedServerPort)))
+            .as("createIngress succeeds")
+            .withFailMessage(String.format("failed to create an ingress for domain %s in namespace %s",
+                domainUid, domainNamespace))
+            .isTrue();
+
+    // check the ingress is created
+    String ingressName = domainUid + "-nginx";
+    assertThat(assertDoesNotThrow(() -> getIngressList(domainNamespace)))
+        .as(String.format("found the ingress %s in namespace %s", ingressName, domainNamespace))
+        .withFailMessage(String.format("can not find ingress %s in namespace %s", ingressName, domainNamespace))
+        .contains(ingressName);
+
+    logger.info("ingress is created in namespace {0}", domainNamespace);
+  }
+
+  @Test
+  @Order(3)
+  @DisplayName("Verify the application can be accessed through the ingress controller")
+  public void testSampleAppThroughIngressController() {
+
+    List<String> managedServerNames = new ArrayList<>();
+    for (int i = 1; i <= replicaCount; i++) {
+      managedServerNames.add(managedServerNameBase + i);
+    }
+
+    // check that NGINX can access the sample apps from all managed servers in the domain
+    String curlCmd = String.format("curl --silent --noproxy '*' -H 'host: %s' http://%s:%s/sample-war/index.jsp",
```

Comment:
using --silent here, when curl fails I think you can't see the error message why its failing by using --silent..use --silent along with --showerror and can you make the curl fail and see if you get the actual error message printed to debug..